src/MensaWidget/MWApiWrapper.class.st

```diff
@@ -22,16 +31,16 @@ MWApiWrapper class >> baseEndpoint [
 
 {
 	#category : #call,
-	#'squeak_changestamp' : 'Louis Christ 6/4/2024 17:23'
+	#'squeak_changestamp' : 'Max 6/18/2024 18:28'
 }
-MWApiWrapper class >> getMealsFrom: anOutletId on: aDate [
+MWApiWrapper class >> getMealsFrom: anOutletId on: aDate using: anApi [
```

Comment:
idk. if "anApi" is a fitting name for that. What do you think about "aWebClient" or "Socket"?

Reply:
I just took inspiration from an existing test example Tom provided, but i am fine with renaming it